assets/src/admin/site-scan-notice/notice.js

```diff
@@ -71,13 +71,15 @@ export function SiteScanNotice() {
 		let elements = [
 			pluginsWithAmpIncompatibility.length > 0 ? (
 				<PluginsWithAmpIncompatibility
+					key={pluginsWithAmpIncompatibility.length}
 					pluginsWithAmpIncompatibility={
 						pluginsWithAmpIncompatibility
 					}
 				/>
 			) : null,
 			themesWithAmpIncompatibility.length > 0 ? (
 				<ThemesWithAmpIncompatibility
+					key={themesWithAmpIncompatibility.length}
```

Comment:
Are these the right keys? The lengths could be the same.

Reply:
Would this be better?

```suggestion
					key="themes"
```